qui/tray/domains.py

```diff
@@ -537,6 +541,7 @@ def __init__(self, app_name, qapp, dispatcher, stats_dispatcher):
             _('<b>Qubes Domains</b>\nView and manage running domains.'))
 
         self.tray_menu = Gtk.Menu()
+        self.widget_icon.connect('focus-out-event', self.hide_menu)
```

Comment:
```
Nov 25 20:30:55.566508 dom0 widget-wrapper[3696]: Traceback (most recent call last):
Nov 25 20:30:55.567032 dom0 widget-wrapper[3696]:   File "/usr/bin/qui-domains", line 33, in <module>
Nov 25 20:30:55.567032 dom0 widget-wrapper[3696]:     sys.exit(load_entry_point('qui==0.1', 'gui_scripts', 'qui-domains')())
Nov 25 20:30:55.567032 dom0 widget-wrapper[3696]:              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
Nov 25 20:30:55.567032 dom0 widget-wrapper[3696]:   File "/usr/lib/python3.13/site-packages/qui/tray/domains.py", line 927, in main
Nov 25 20:30:55.567032 dom0 widget-wrapper[3696]:     app = DomainTray(
Nov 25 20:30:55.567032 dom0 widget-wrapper[3696]:         'org.qubes.qui.tray.Domains', qapp, dispatcher, stats_dispatcher)
Nov 25 20:30:55.567032 dom0 widget-wrapper[3696]:   File "/usr/lib/python3.13/site-packages/qui/tray/domains.py", line 544, in __init__
Nov 25 20:30:55.567032 dom0 widget-wrapper[3696]:     self.widget_icon.connect('focus-out-event', self.hide_menu)
Nov 25 20:30:55.567032 dom0 widget-wrapper[3696]:     ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Nov 25 20:30:55.567032 dom0 widget-wrapper[3696]: TypeError: <Gtk.StatusIcon object at 0x722941d2e000 (GtkStatusIcon at 0x5e0a8c5d4f30)>: unknown signal name: focus-out-event
Nov 25 20:30:55.597149 dom0 widget-wrapper[3695]: exiting with 1
```

Reply:
This should work if you connect it to the self.tray_menu, but I only tested it on X, not on Wayland.